src/WebJobs.Script.Analyzers/Constants.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
```

Comment:
Can you ensure all the new files have the appropriate copyright header/info?

Reply:
Done.